libsolidity/formal/SymbolicTypes.cpp

```diff
@@ -544,17 +544,9 @@ void setSymbolicUnknownValue(smtutil::Expression _expr, frontend::TypePointer co
 smtutil::Expression symbolicUnknownConstraints(smtutil::Expression _expr, frontend::TypePointer const& _type)
 {
 	solAssert(_type, "");
-	if (isEnum(*_type))
+	if (isEnum(*_type) || isInteger(*_type) || isAddress(*_type))
```

Comment:
Can you also add fixed bytes

Reply:
I will, just trying to find an example where the problem would show up for `FixedBytes`.

Reply:
It seems that the problem that shows up for `msg.sender` does not occur when there is a local variable of type `Address` or `FixedBytes`, because there the constraints are deduced from the type of the *symbolic* variable, and there the type is recorded as `IntType` already, even if the corresponding Soldity variable has type `Address` or `FixedBytes`.
So the constraints were missing only for the transaction data.